.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    "es2015"
+    "@babel/preset-env"
```

Comment:
> https://babeljs.io/docs/en/babel-preset-env#no-targets
> Since one of the original goals of preset-env was to help users easily transition from using preset-latest, it behaves similarly when no targets are specified: preset-env will transform all ES2015-ES2020 code to be ES5 compatible.

との記載がありますので`package.json`で`@babel/preset-env`を使用した場合はこのオプションを指定するでよいかと思います。